### ISSUE TITLE
Is Encryptr still actively developed?

### ISSUE THREAD

**Issue by dermitdemdino** [SpiderOak/Encryptr#296]
Unfortunately, the development seems to be resting. The last Android version is from 2015 (!), You can still not import / export.

**Comment by Samahedron** [comment]
I just put this question to Spideroak customer service. Their response below:

 Kat Tyler (SpiderOak Support)

Sep 3, 1:32 PM CDT

Hi [Samahedron],

I'm Kat with SpiderOak Support. Thanks for contacting us!

Our developers are currently working on a new version of Encryptr, which should address the issues some Android users have been seeing when trying to load the application. I'm unfortunately unaware of an ETA on when that might be released - we try not to release new versions until they're completely ready and have been thoroughly tested.

That new version will be the last release for this iteration of the product, as we're end-of-lifing Encryptr as it exists now. That new version will include an export feature, but won't receive further updates or added features. Encryptr may come back in a new form at some point in the future, but I don't have concrete details on if or when that might happen.

I wish I had better news for you, or could even give you a timeframe to work with, but I've shared with you everything I know about the current state of Encryptr. If there's anything more we can do, please let me know.

Thanks,
Kat

**Comment by lowellday** [comment]
If Encryptr has reached end-of-life **PLEASE** make a formal announcement and provide users with a way to export their data!

**Comment by devgeeks** [comment]
@lowellday I cannot speak to formal announcements, but I believe SpiderOak are working on releasing an update to Encryptr to allow export of data. I assume for the purpose of migrating off Encryptr :(

**Comment by 4oo4** [comment]
@devgeeks Really appreciate all the work you've put into this amazing app and the recent updates, but shame on SpiderOak for killing off the project and for creating this air of confusion.

**Comment by deepdivedylan** [comment]
SpiderOak messaged me on Twitter that should be added to this thread.

**TL; DR:** Encryptr is being moved to the Flow platform, and not going EOL as previously posted on these threads.

https://twitter.com/deepdivedylan/status/918210143606669315

>It's been a while since an update, but they are coming! A complete rewrite will be out early next year too.
>
>That work has already started and is progressing nicely, but we’re nearing an intermediate release with export and offline mode.
>
>We are migrating Encryptr to the Flow platform (which Semaphor is built on) and will continue developing it as a SpiderOak product.
>
>Right, we need to put a formal announcement there. For a time the plan was to EoL it, but that’s not true now.

**Comment by wallyxie** [comment]
@deepdivedylan Kind of late reading this, but this is really good to hear. Thanks for the update.

**Comment by julian-higginson** [comment]
It's march already....  Still no android release newer than 2015. 

And you're now well over 6 months with an app that critically fails for a whole bunch of android users. 

Still nothing more official than some 5 month old tweets about migrating to a new platform "ANY DAY NOW" and how you "really need to get some announcements up" 

Lets just say, it's not only looking bad for Encryptr right now, it's looking very bad for Spideroak.

**Comment by 23RoMax** [comment]
As @deepdivedylan already wrote, they are still in the process of porting it.

I contacted them via Twitter, they are very responsive there.
Q3 is a possible ETA of the port to the new platform.

https://twitter.com/SpiderOak/status/980858635621462019

**Comment by heyakyra** [comment]
Any day now, eh?

**Comment by Samahedron** [comment]
https://twitter.com/SpiderOak/status/1019683986539458561

> @spideroak:  thanks for your question, we don't have a specific release date, but we'll be sure to notify you when it's ready.

**Comment by JulianMeans** [comment]
well, it sure looks like "early 2018" for a rewrite of the application has blown out a bit. it's now early 2019.... 

Funnily enough, my encryptr has started working on my phone at some point (the same phone it had STOPPED working on a year and a half ago) but there's no new app, still. The latest release of encryptr on google play is still dated 2015...

**Comment by psivesely** [comment]
Yeah, when I have time I'm planning on forking it just to implement export to CSV. It's still working for me, but as far as I'm concerned this software is dead and there are better e2ee cloud-syncing password managers anyway. I'll ping here and on #278, #316, #278, #159 if I get around to it before someone else does. I'm stuck with Encryptr for now, but it still gets the basic job done, so it's not super pressing for me.

**Comment by dannyyu92** [comment]
@nvesely any recommendations on other e2ee cloud-syncing password managers?

**Comment by heyakyra** [comment]
@bitwarden https://bitwarden.com/

**Comment by deepdivedylan** [comment]
@nvesely the version of Encryptr on `master` already has CSV export. The instructions on how to build a version of Encryptr with CSV export are [on another thread](https://github.com/SpiderOak/Encryptr/issues/295#issuecomment-322449705). The process is awkward, but it worked for me to export and migrate to another password manager. 😺

**Comment by student237** [comment]
Today I finished migrating my passwords from Encryptr to Bitwarden. Thanks to everyone for the password manager suggestions, and to @devgeeks and others for the work they have put into Encryptr over the years.

**Comment by royaleconqueso** [comment]
This is from last month:
https://twitter.com/SpiderOak/status/1240696041885687810
Am I reading too much into an "export" feature, when I think it's about to be killed off?
